### ISSUE TITLE
Clarify use of "tag" in PDF14

### ISSUE THREAD

**Issue by CharlesBelov** [w3c/wcag#3197]
WCAG 2.0 technique [PDF14: Providing running headers and footers in PDF documents](https://www.w3.org/TR/WCAG20-TECHS/PDF14.html) refers to artifact tags, and to the idea that if headers and footers are properly created in Word, they will be carried over to PDF documents marked up as artifacts.

First, it is not clear whether an artifact tag is an accessibility tag. When I convert a Word document having correctly created headers and footers into a tagged PDF, I do not see artifact tags in the Accessibility Tag list. I do see the headers and footers in the document itself. When I try to insert an artifact tag into the accessibility tag pane, I do not see Artifact in the list of available tags. If an artifact target is a different type of tag, it would be an improvement to this page to clarify that.

Second, it is not made clear that, because artifacts are (apparently) not accessibility tags, artifacts are not read aloud by screen readers. It would be good to make that clear.

Third, it would be good to add as a related technique [PDF17: Specifying consistent page numbering for PDF documents](https://www.w3.org/TR/WCAG20-TECHS/PDF17.html).

**Comment by alastc** [comment]
Discussion from the friday meeting: The intro is quite confusing because it talks about "providing" headers & footers, but artifacts are hidden from the tag structure. Lori/Francis will check this issue when they get to PDF14.

**Comment by ljoakley** [comment]
@CharlesBelov PDF14 is simply for adding headers and footers to a document that will be exported to a PDF. A discussion of artifacts and tags could be added to another technique, not this one.

**Comment by DuffJohnson** [comment]
I'm unclear as to why this technique was created in the first place. There's not much of an "accessibility story" with headers and footers, not least because, as @alastc notes, headers and footers rarely include interactive navigation features, are typically marked as artifact, and are thus not included in the tag tree, so they are functionally invisible to AT users who rely on tags.

Of course, it's fine to recommend headers / footers on longer / sectioned documents for their putative benefit for users with cognitive disabilities, but this is only a subset of documents, and should not be a general recommendation.

With respect to the SC this technique purportedly supports...

- SC 2.4.8: Given that correct handling of headers / footers is to mark them as artifacts this technique ONLY pertains to SC 2.4.8 for users who **don't use** AT.
- SC 3.2.3: Page headers / footers in PDF barely qualify as "navigation" because they generally don't include actionable navigation elements. Page "identification" would be more correct than "navigation"... and in any event, headers / footers only represent identification / navigation to users who consume the document visually, without reference to the tags.

I note that [PDF4](https://www.w3.org/WAI/WCAG22/Techniques/pdf/PDF4)), while about cosmetic images, does address artifacts in general terms, and includes reference to page headers / footers.

As written this technique seems to add little / no value, is confusing, and therefore IMHO may be readily dropped.

**Comment by mraccess77** [comment]
I do come across interactive links to be in the footer of PDFs.  Headers and footers also contain important markings such as confidential and other classifications - so artificing them needs to be done with careful consideration.

**Comment by DuffJohnson** [comment]
Totally agree! However, this point isn't about headers and footers per se, but about the distinction between what we in PDF-land call "real content" (which must be tagged) vs. "artifact content" (which must be marked as Artifact). 

A wrinkle here is that authoring software (AFAIK) tends not to support instances of real content in page headers and footers, leaving it up to remediators to ensure accessibility. This is another reason for WCAG to eschew recommending page headers / footers as an "accessibility technique" (at least until authoring software gets smarter).

**Comment by CharlesBelov** [comment]
The accessibility issue I've encountered is when content creators put information in the heading that is not in the document. For instance, they cast a document title as a heading on page one and it isn't anywhere in the document tags.

**Comment by CharlesBelov** [comment]
Also, there's an issue with Excel-to-PDF. If you put the sheet title in the heading, it's not available to assistive technology, so people using screen readers can't tell which sheet they're on in a multi-sheet PDF. But if you put the sheet title in the first row of the table, that messes up reading column headers as one moves across a row from data cell to data cell.

**Comment by DuffJohnson** [comment]
@CharlesBelov I suppose the first problem you mention could be addressed in this technique by improving the description to make it abundantly clear that the use of page headers / footers (**not** "headings") has the effect of removing such content from the tag tree. That would, however, leave a very confused technique (recommending the use of headers while clarifying that such content doesn't work with AT). This is why I suggest dropping this technique entirely.

The Excel-to-PDF problem is purely an implementation thing. The vendor could do better, and should be encouraged to do so...
